npm/webpack-dev-server/src/loader.ts

```diff
@@ -16,7 +17,7 @@ const makeImport = (file: Cypress.Cypress['spec'], filename: string, chunkName:
   return `"${filename}": {
     shouldLoad: () => document.location.pathname.includes(${JSON.stringify(file.relative)}),
     load: () => {
-      return import(${JSON.stringify(path.resolve(projectRoot, file.relative), null, 2)} ${magicComments})
+      return import(${JSON.stringify(path.resolve(projectRoot, file.relative))} ${magicComments})
```

Comment:
The support file needs a similar line to `path.resolve(projectRoot, support)` before we JSON stringify it in the import.